pypit/traceslits.py

```diff
@@ -155,6 +155,7 @@ def __init__(self, mstrace, pixlocn, binbpx=None, settings=None, det=None,
         self.lcen = None     # narray
         self.rcen = None     # narray
         self.tc_dict = None  # dict
+        # TODO Why is there a tc_dict and a tslits_dict. This is confusing naming without some docs or comments here.
```

Comment:
These two are completely different.

```tc_dict``` is defined in the doc string (albeit not in great detail)